### PR TITLE
Tag Swifter v0.0.7

### PR DESCRIPTION
**Pull request by wookay**
nice to meet everyone at JuliaCon 2016!

Updated followings
- fixed to show query result
- displaying views on the jupyter notebook for julia 0.4
- added `valueof` to use the result value
- specified the build number

### CODE REVIEW

File: Swifter/versions/0.0.7/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Requests
+JSON 0.0 0.6
```
Comment (tkelman) [inline Swifter/versions/0.0.7/requires]: do you still need this upper bound or was this only a test issue?

Comment (wookay) [inline Swifter/versions/0.0.7/requires]: oh, it was a test issue, just copied from 0.0.6's one.
I'll change that to be just JSON.

Comment (wookay) [inline Swifter/versions/0.0.7/requires]: meanwhile, should I change all older ones (from 0.0.1 to 0.0.6) also to be just JSON?

Comment (tkelman) [inline Swifter/versions/0.0.7/requires]: you haven't retroactively fixed the test in past versions, have you?

Comment (wookay) [inline Swifter/versions/0.0.7/requires]: right, I haven't changed anything, so I suppose to be remaining past versions as "JSON 0.0 0.6".

Comment (tkelman) [inline Swifter/versions/0.0.7/requires]: requests or some other package is still holding your version of JSON back to 0.5.4, but I checked with `Pkg.checkout("Swifter"); Pkg.checkout("JSON"); Pkg.test("Swifter")` and it all looks good.

Comment (wookay) [inline Swifter/versions/0.0.7/requires]: thank you very much!

**Pull request merged**